LTI-JML (Ejercicio ATS)

moncio opened the pull request:
Analisis y diseño de un sistema de gestion de candidatos utilizando asistentes IA

alvarotech commented:
Muy buen trabajo William!

- Muy buena propuesta de producto, con elementos innovadores en varios pasos del customer journey (IA, analisis predictivo...)
- Lean Canvas: buen intento de diagrama usando elementos visuales, aquí hay que tirar de creatividad ya que mermaid no está adaptado para este diagrama
- El modelo de datos está bien (nombre y tipo de campo, PKs y FKs...) e incluye entidades para representar a 2 actores principales, los usuarios internos de la aplicación, recruiters y managers, así como las tareas.
- Buen diseño de sistema y arquitectura de microservicios, es concreto y entra en detalle de tecnologías. En cualquier caso, echo de menos componentes como monitoreo y balanceo de carga, así como otros elementos básicos de cualquier sistema en la nube (CDN para el frontend?)
El diagrama C4 está bien pero me faltan los diagramas, y le falta el nivel 4, detalles a nivel código de alguna de las clases principales del servicio de gestión de candidatos

moncio committed:
Ejercicio Applicant-Tracking System

Analisis y diseño de un sistema inteligente de candidatos para trabajo utilizando asistentes IA

moncio committed:
Actualizado prompts.md
